test/e2e/commands/xmlupload/test_list_client_live.py

```diff
@@ -1,17 +1,13 @@
 import regex
 
 from dsp_tools.commands.xmlupload.list_client import ListClientLive
-from dsp_tools.commands.xmlupload.project_client import ProjectClientLive
 from dsp_tools.utils.connection_live import ConnectionLive
 
 
 def test_list_client_live() -> None:
     con = ConnectionLive("http://localhost:3333")
     con.login("root@example.com", "test")
-    project_client = ProjectClientLive(con, "0001")
-    project_iri = project_client.get_project_iri()
-    assert project_iri == "http://rdfh.ch/projects/0001"
-    list_client = ListClientLive(con, project_iri)
```

Comment:
project_client.get_project_iri() is already tested in test_project_client_live.py